state/protocol/badger/mutator.go

```diff
@@ -32,11 +33,13 @@ import (
 // the non-consensus nodes have to perform.
 type FollowerState struct {
 	*State
-	index    storage.Index
-	payloads storage.Payloads
-	tracer   module.Tracer
-	consumer protocol.Consumer
-	cfg      Config
+
+	index             storage.Index
+	payloads          storage.Payloads
+	tracer            module.Tracer
+	consumer          protocol.Consumer
+	cfg               Config
+	complianceMetrics module.ComplianceMetrics
```

Comment:
This is already accessible via the embedded `State` -- don't need to add it again here